src/Numerics/project.json

```diff
@@ -0,0 +1,97 @@
+﻿{
+  "name": "MathNet.Numerics.Core",
```

Comment:
If possible we should avoid to split the ecosystem even further. This PR is adding .Net Standard support exactly to avoid splitting between .Net and .Net Core unnecessarily. I think we should try to have just one package supporting everything, without the Core suffix.